core/src/worker/api.ts

```diff
@@ -134,3 +135,21 @@ export async function getAggregator({
     throw new OraklError(OraklErrorCode.FailedToGetAggregator)
   }
 }
+
+/**
+ * Store catched RR worker error log to
+ * `Orakl Network API` `error` endpoint
+ *
+ * @param {data} IErrorMsgData
+ * @param {Logger} logger
+ * @exception {FailedToGetAggregate}
+ */
+export async function storeErrorMsg({ data, logger }: { data: IErrorMsgData; logger: Logger }) {
+  try {
+    const error = (await axios.post(ERROR_ENDPOINT, data))?.data
+    console.dir(error, { depth: null })
```

Comment:
I think we don't need this log. The error is already logger by pino in the parent call.